Description: When building the package, the source is not necessarily part
 of a git repository. This patch removes all git command executions. Additionally
 it replaces GIT_REVISION with the current Debian version number. This will go into
 some of the firmware files.
Author: Ruben Undheim <ruben.undheim@gmail.com>

Index: ubertooth/firmware/common.mk
===================================================================
--- ubertooth.orig/firmware/common.mk	2017-01-22 11:22:35.631519826 +0100
+++ ubertooth/firmware/common.mk	2017-01-22 11:22:35.627519782 +0100
@@ -52,13 +52,13 @@
 	UBERTOOTH_OPTS += -DTX_ENABLE
 endif
 
-DIRTY = $(shell git status -s --untracked-files=no)
-ifneq ($(DIRTY), )
-	DIRTY_FLAG = *
-endif
+#DIRTY = $(shell git status -s --untracked-files=no)
+#ifneq ($(DIRTY), )
+#	DIRTY_FLAG = *
+#endif
 
 # automatic git version when working out of git
-GIT_REVISION ?= -D'GIT_REVISION="git-$(shell git log --pretty=format:'%h' -n 1)$(DIRTY_FLAG)"'
+GIT_REVISION ?= -D'GIT_REVISION="2015.09.R2-1"'
 
 # compile info
 COMPILE_BY ?= -D'COMPILE_BY="$(shell whoami)"'
